dropwizard-jetty/src/test/java/io/dropwizard/jetty/ContextRoutingHandlerTest.java

```diff
@@ -66,6 +66,9 @@ public void passesHandlingNonMatchingRequests() throws Exception {
 
     @Test
     public void startsAndStopsAllHandlers() throws Exception {
+        when(handler1.isStopped()).thenReturn(true);
+        when(handler2.isStopped()).thenReturn(true);
```

Comment:
Accounts for https://github.com/eclipse/jetty.project/commit/6c1b5982a5577d6110ce8c985424f823673bd6cc